src/plugins/maps_dashboards/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build
```

Comment:
should we add node modules? or can we just copy this file from another plugin so that we don't need to think about it going forward?

Reply:
Yes, will update this!